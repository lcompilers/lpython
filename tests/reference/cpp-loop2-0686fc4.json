{
    "basename": "cpp-loop2-0686fc4",
    "cmd": "lpython --no-color --show-cpp {infile}",
    "infile": "tests/loop2.py",
    "infile_hash": "7946c522ceb16f99810780d4aba7fa2593695a4b49fb35ea1f131f53",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "cpp-loop2-0686fc4.stdout",
    "stdout_hash": "1cb26a16bf52329e023b6b903e2d280d402ec6963da181cdd3505718",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}