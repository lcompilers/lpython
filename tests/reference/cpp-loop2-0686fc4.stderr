warning: The module 'sys' located in $DIR\src\bin/../runtime/sys.py cannot be loaded
 --> tests/loop2.py:1:1
  |
1 | from sys import exit
  | ^^^^^^^^^^^^^^^^^^^^ imported here

semantic error: Variable 'error_code' not declared
 --> $DIR\src\bin/../runtime/sys.py:8:10
  |
8 |     quit(error_code)
  |          ^^^^^^^^^^ 
